cardano-node/src/Cardano/Node/Configuration/POM.hs

```diff
@@ -329,7 +332,8 @@ instance FromJSON PartialNodeConfiguration where
 
       -- Genesis syncing protocol
       -- DISABLED BY DEFAULT
-      pncEnableGenesis <- Last <$> v .:? "EnableGenesis" .!= Just False
+      pncEnableGenesis      <- Last <$> v .:? "EnableGenesis" .!= Just False
```

Comment:
Why two configuration values? If the genesis config plans aren't present then we know to not enable it.

Reply:
Thanks, what would you prefer instead? Something like
```yaml
Genesis:
  Enable: True
  # optionally
  LowLevelOptions: 
    ....
```
?